cmd/fyne/internal/commands/build.go

```diff
@@ -224,20 +224,25 @@ func (b *Builder) build() error {
 		appendEnv(&env, "CGO_LDFLAGS", "-mmacosx-version-min=10.11")
 	}
 
+	ldFlags := ""
```

Comment:
Feels a little simpler if this was `ldFlags := extractLdFlags(...)`?
It avoids the somewhat confusing named `updateGoLdFlags` and we don't have to merge two sources of ld flags.
If `extractLdFlags` updated GOFLAGS to complete the extract it could be done without the extra method.

Reply:
Removing extractLdFlags and merging it in extractLdflagsFromGoFlags would make that later function more complex and more difficult to test. Still simplifying build to first extract the ldflags from GOFLAGS make sense and simplify the code.